opentelemetry-sdk/src/opentelemetry/sdk/trace/sampling.py

```diff
@@ -413,7 +456,7 @@ class _ParentBasedAlwaysOn(ParentBased):
     def __init__(self, _):
         super().__init__(ALWAYS_ON)
 
-
+# TODO(sconover): Something must be done to allow samplers from the contrib repo to be specified
```

Comment:
The most immediate issue is what to do about `_KNOWN_SAMPLERS` - verification of sampler choice based on this dict excludes the possibility of contrib-repo samplers, I would appreciate advice about what to do about this.